test/module/shared_model/builders/protobuf/transaction_responses/proto_transaction_response_factory.cpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2018 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include "framework/specified_visitor.hpp"
+
```

Comment:
shouldn't it be with the following spacing/ordering?:
```
#include "backend/protobuf/proto_tx_status_factory.hpp"

#include <gtest/gtest.h>

#include "framework/specified_visitor.hpp"
```

Reply:
No, google codestyle defines the following order:
- related header (`proto_tx_status_factory.hpp)` here
- empty line
- system headers (`iostream`, `vector`, ...)
- empty line
- other headers (custom dependencies such as `boost`, `gtest`, and other project headers)

Reply:
I have different [info](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)
![image](https://user-images.githubusercontent.com/5658339/45436759-2b380a80-b6bc-11e8-920b-58201768dbf2.png)

Reply:
This policy a bit complicated. I guess it definitely should be in CI checks.